src/passes/InstrumentLocals.cpp

```diff
@@ -57,15 +57,15 @@ Name get_f32("get_f32");
 Name get_f64("get_f64");
 Name get_v128("get_v128");
 Name get_funcref("get_funcref");
-Name get_anyref("get_anyref");
+Name get_extref("get_extref");
```

Comment:
We don't abbreviate `externref` to `extref` in other places, so how about just `get_externref`/`set_externref` here as well?